src/cli.js

```diff
@@ -1,6 +1,20 @@
 import * as jsApi from './node-jspm-jasmine.js';
+import commander from 'commander';
 
-jsApi.runTests({}, function(err) {
-	console.error(err);
-	process.exit(1);
-});
+export function run(args) {
+
+    commander
+        .option('-c, --jasmineConfig <path>')
```

Comment:
Could you change this to `--jasmine-config`? I think that's the standard for CLIs